sdk/mixedreality/azure-mixedreality-remoterendering/README.md

```diff
@@ -0,0 +1,392 @@
+        .setInputRelativeAssetPath("box.fbx")
+        .setOutputStorageContainerUrl(storageURL);
+    
+    // A randomly generated UUID is a good choice for a conversionId.
+    String conversionId = UUID.randomUUID().toString();
+    
+    SyncPoller<AssetConversion, AssetConversion> conversionOperation = client.beginConversion(conversionId, conversionOptions);
+```
+
+The output files will be placed beside the input asset.
+
+### Convert a more complex asset
+
+Assets can reference other files, and blob containers can contain files belonging to many different assets.
+In this example, we show how prefixes can be used to organize your blobs and how to convert an asset to take account of that organization.
+Assume that the blob container at `inputStorageURL` contains many files, including "Bicycle/bicycle.gltf", "Bicycle/bicycle.bin" and "Bicycle/saddleTexture.jpg".
+(So the prefix "Bicycle" is acting very like a folder.)
+We want to convert the gltf so that it has access to the other files which share the prefix, without requiring the conversion service to access any other files.
+To keep things tidy, we also want the output files to be written to a different storage container and given a common prefix: "ConvertedBicycle".
+The code is as follows:
+
+```
+    AssetConversionOptions conversionOptions = new AssetConversionOptions()
+        .setInputStorageContainerUrl(inputStorageURL)
+        .setInputRelativeAssetPath("bicycle.gltf")
+        .setInputBlobPrefix("Bicycle")
+        .setOutputStorageContainerUrl(outputStorageURL)
+        .setOutputBlobPrefix("ConvertedBicycle");
+    
+    String conversionId = UUID.randomUUID().toString();
+    
+    SyncPoller<AssetConversion, AssetConversion> conversionOperation = client.beginConversion(conversionId, conversionOptions);
+```
+
+> NOTE: when a prefix is given in the input options, then the input file parameter is assumed to be relative to that prefix.
+> The same applies to the output file parameter in output options.
+
+### Get the output when an asset conversion has finished
+
+Converting an asset can take anywhere from seconds to hours.
+This code uses an existing conversionOperation and polls regularly until the conversion has finished or failed.
+The default polling period is 10 seconds.
+Note that a conversionOperation can be constructed from the conversionId of an existing conversion and a client.
+
+```
+    AssetConversion conversion = conversionOperation.getFinalResult();
+    if (conversion.getStatus() == AssetConversionStatus.SUCCEEDED)
+    {
+        logger.info("Conversion succeeded: Output written to {}", conversion.getOutputAssetUrl());
+    }
+    else if (conversion.getStatus() == AssetConversionStatus.FAILED)
+    {
+        logger.error("Conversion failed: {} {}", conversion.getError().getCode(), conversion.getError().getMessage());
+    }
+    else
+    {
+        logger.error("Unexpected conversion status: {}", conversion.getStatus());
+    }
+```
```

Comment:
Nit: we should not put opening braces on new lines, and we should put the following `else if` and `else` statements on the same line as the closing brace. The format above feels very C# and not very Java. Same comment applies elsewhere in this document.

Reply:
I am addressing these as part of using proper code snippets (as per Srikanta's comment above)